docs/provision-with-machine.md

```diff
@@ -0,0 +1,174 @@
+
+```
+docker-machine create -d virtualbox local
+```
+
+This examples uses VirtualBox but it could easily be DigitalOcean or a host on
+your data center. The `local` value is the host name. Once you create it,
+configure your terminal's shell environment to interact with the `local` host.
+
+```
+eval "$(docker-machine env local)"
+```
+
+Each Swarm host has a token installed into its Engine configuration. The token
+allows the Swarm discovery backend to recognize a node as belonging to a
+particular Swarm cluster. Create the token for your cluster by running the
+`swarm` image:
+
+```
+docker run swarm create
+Unable to find image 'swarm' locally
+1.1.0-rc2: Pulling from library/swarm
+892cb307750a: Pull complete
+fe3c9860e6d5: Pull complete
+cc01ef3f1fbc: Pull complete
+b7e14a9c9c72: Pull complete
+3ec746117013: Pull complete
+703cb7acfce6: Pull complete
+d4f6bb678158: Pull complete
+2ad500e1bf96: Pull complete
+Digest: sha256:f02993cd1afd86b399f35dc7ca0240969e971c92b0232a8839cf17a37d6e7009
+Status: Downloaded newer image for swarm
+0de84fa62a1d9e9cc2156111f63ac31f
+```
+
+The output of the `swarm create` command is a cluster token. Copy the token to a
+safe place you will remember. Once you have the token, you can provision the
+Swarm nodes and join them to the cluster_id.  The rest of this documentation,
+refers to this token as the `SWARM_CLUSTER_TOKEN`.
+
+## Provision Swarm nodes
+
+All Swarm nodes in a cluster must have Engine installed. With Machine and the `SWARM_CLUSTER_TOKEN` you can provision a host with Engine and configure it as a Swarm node with one Machine command. To create a Swarm master node on a new VM called `swarm-master`, you do the following:
+
+```
+docker-machine create \
+    -d virtualbox \
+    --swarm \
+    --swarm-master \
+    --swarm-discovery token://SWARM_CLUSTER_TOKEN \
+    swarm-master
+```
+
+Then, provision additional an additional node. You must supply the `SWARM_CLUSTER_TOKEN` and a unique name for each host node, `HOST_NODE_NAME`.
+
+```
+docker-machine create \
+    -d virtualbox \
+    --swarm \
+    --swarm-discovery token://SWARM_CLUSTER_TOKEN \
+    HOST_NODE_NAME
+```
+
+For example, you might use `node-01` as the `HOST_NODE_NAME` in the previous example.
+
+>**Note**: These command rely on Docker Swarm's hosted discovery service, Docker
+Hub. If Docker Hub is having issues, these commands may fail. You can wait for
+the Docker Hub to recover or configure other types of discovery backends. Check
+the [Docker Hub status page](http://status.docker.com/) for service
+availability.
+
+## Connect node environments with Machine
+
+If you are connecting to typical host environment with Machine, you use the `env` subcommand, like this:
+
+```
+eval "$(docker-machine env local)"
+```
+
+Docker Machine provides a special `--swarm` flag with its `env` command to connect to Swarm nodes.
+
+```
+docker-machine env --swarm HOST_NODE_NAME
```

Comment:
@MHBauer This is syntax.  So,  I don't provide output for that.  The real issue is the previous command which does show a node name.